### PR TITLE
dropdown: improve JAWS readability

### LINKED ISSUE #1501: dropdown: improve JAWS readability
JAWS should read
- [x] `dropdown` as "button-menu collapsed"
- [x] `_disabled` as "unavailable"
- [x] `dropdown_opened` as "button-menu expanded"
- [x] `popup` field of `dropdown_opened` as corresponding `popup` block
- [x] `switcher` field

Check and fix divergencies.

### PR DESCRIPTION
**Pull request by sipayRT**
Closes #1501

### CODE REVIEW

**Comment by sipayRT** [comment]
@aristov need your opinion here

**Comment by aristov** [comment]
#1393 was merged. Rebase, please.

**Comment by sipayRT** [comment]
@aristov gemini-tests fails because of focus changing. I'll fix screenshots after your review

File: common.blocks/dropdown/dropdown.js [hunk 1]
```diff
@@ -31,8 +31,20 @@ provide(BEMDOM.decl(this.name, /** @lends dropdown.prototype */{
...
-        'opened' : function(_, modVal) {
-            this.getPopup().setMod('visible', modVal);
+        'opened' : {
+            '*' : function(_, modVal) {
+               this.getPopup().setMod('visible', modVal);
+               this._switcher.domElem.attr('aria-expanded', !!modVal);
+           },
+
+           'true' : function() {
+               this._popup.domElem.attr('tabindex', 0).focus();
```
Comment (aristov) [inline common.blocks/dropdown/dropdown.js]: This solution contradicts our a11y manual: https://nda.ya.ru/3Quhd7

> Focus should be on button after dropdown expansion.

Let's solve focus maintaining problems later and separately from readability (#1026).

**Comment by aristov** [comment]
LGTM

**Pull request merged**